pages/events/_id/edit.vue

```diff
@@ -17,6 +17,7 @@
         vertical
         :show-remove="!!item.id"
         :show-copy="!!item.id"
+        :event-error="eventError"
```

Comment:
Form component shouldn't know about event

Reply:
@razbakov 
What does it mean?
`form component shouldn't know about event`

Reply:
`TForm` is a component that renders forms. 
Event is a data structure.

Abstract component that renders forms should operate with abstract concepts like fields, errors, etc.

TForm component is used in Profile Editor, Event Editor, Post Editor. Following naming logic with `event-error` we would need to add also `profile-error`, `post-error`. That solution is not scalable.